src/language-server/workspace.ts

```diff
@@ -215,9 +215,6 @@ export class GraphQLWorkspace {
     }
     // If project exists, update the config
     if (project && config) {
-      if (equal(project.config.rawConfig, config.rawConfig)) {
-        return;
-      }
```

Comment:
I added this logic when I saw that there were too many file events happening, but didn't know why and assumed the reason was something outside the scope of this Extension.

Removing it here again, since sometimes we actually want a restart even if the config didn't really change.